### PR TITLE
Configurable holidays

### PR DESCRIPTION
**Pull request by bluec**
Added four configurable holiday lists for greater flexibility. This allows the user to define their own holidays instead of only using those available from the gov.uk json feed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Ignore common IDE files
+/.idea
```
Comment (bobbyshaw) [inline .gitignore]: We recommend putting IDE files in your global git ignore (https://help.github.com/articles/ignoring-files/)

**Review by bobbyshaw — CHANGES_REQUESTED** [review]
Thanks @bluec, this looks useful!

File: .gitignore (hunk 1 above)
Comment (bluec) [reply .gitignore]: Thanks Tom I've no idea why I don't do that. I already put .DS_Store and others in there; the thought just never occured to me!

Comment (bluec) [reply .gitignore]: Just let me figure out how on earth to accept this change request. Do I just create a new PR? The github UI is like a labyrinth!

**Review by bluec — COMMENTED** [review]
@bobbyshaw Have removed .gitignore thanks for the feedback

**Review by bobbyshaw — APPROVED** [review]

**Comment by bobbyshaw** [comment]
Thanks for the update @bluec. I've merged, I've changed base to the develop branch to try and keep to gitflow methodology.

**Comment by bluec** [comment]
Hi, could this be merged into master and released? Seems develop and master have got a bit out of sync?

**Comment by punkstar** [comment]
@bluec Sorted.

**Pull request merged**